wandb/sdk/launch/_project_spec.py

```diff
@@ -370,6 +374,7 @@ def create_project_from_spec(launch_spec: Dict[str, Any], api: Api) -> LaunchPro
         launch_spec.get("overrides", {}),
         launch_spec.get("resource", "local"),
         launch_spec.get("resource_args", {}),
+        launch_spec.get("cuda", False),
```

Comment:
When we pull the `wandb-metadata.json` file we can check that for a cuda version if this is false.